admin-portal/src/pages/aidPackage/assignSuppliers/supplierNeedAllocationTable/index.tsx

```diff
@@ -90,7 +90,7 @@ export default function SupplierNeedAllocationTable({
             `${quote.period.day}/${quote.period.month}/${quote.period.year}`
           ).format("MM/DD/YYYY"),
           quantity,
-          max: Math.min(requiredQuantity, quote.availableQuantity),
+          max: quote.remainingQuantity,
```

Comment:
This has to be changed while you are editing quantities on this view itself, right? Therefore we need to calculate the new remaining quantity.